packages/core/src/components/Section/Section.test.tsx

```diff
@@ -0,0 +1,114 @@
+import { fireEvent, render } from "@testing-library/react";
+import { describe, expect, it, vi } from "vitest";
+import { HvSection } from "./Section";
+
+describe("Section", () => {
+  it("should contain all the steps", () => {
+    const { getByText } = render(<HvSection title="Section Title" />);
+    expect(getByText("Section Title")).toBeInTheDocument();
```

Comment:
Using `screen` [is better](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen)
```suggestion
    render(<HvSection title="Section Title" />);
    expect(screen.getByText("Section Title")).toBeInTheDocument();
```